lm_eval/tasks/mmlu_vi/mmlu_vi.yaml

```diff
@@ -0,0 +1,17 @@
+task: mmlu_vi
+dataset_path: vlsp-2023-vllm/mmlu # a copy of `cais/mmlu` with no auxiliary_train split
+test_split: test
+fewshot_split: dev
+fewshot_config:
+  sampler: first_n
+# description": "The following are multiple choice questions (with answers) about ."
+output_type: multiple_choice
+doc_to_text: "The following are multiple choice questions (with answers) about .\n\n{{question.strip()}}\nA. {{choices[0]}}\nB. {{choices[1]}}\nC. {{choices[2]}}\nD. {{choices[3]}}\nAnswer:"
```

Comment:
Would the prompts still be in English? Or would it be better to be in Vietnamese? (Same goes for other tasks here)